docs/apidocs/qiskit_addon_obp.utils.metadata.rst

```diff
@@ -1,10 +1,17 @@
-========
-metadata
-========
-
-.. _qiskit_addon_obp-utils-metadata:
+===========================================================
+Metadata utilities (:mod:`qiskit_addon_obp.utils.metadata`)
+===========================================================
 
 .. automodule:: qiskit_addon_obp.utils.metadata
    :no-members:
    :no-inherited-members:
    :no-special-members:
+
+.. currentmodule:: qiskit_addon_obp.utils.metadata
+
+.. autosummary::
+   :toctree: ../stubs/
+   :nosignatures:
+
+   OBPMetadata
+   SliceMetadata
```

Comment:
I stumbled across this: why did you choose to use `autosummary` here but explicitly list each function/class separetely below (next comment)?

This here will correctly use the `class.rst` template file that we have in place, e.g. setting `:show-inheritance:` etc because each class gets its own subpage.

Reply:
From the PR description:

> Inlines functions and some classes
>
> We've gotten prior user complaints that they don't like when you have to click to a bunch of subpages to see individual functions and classes. So, we've generally adopted the approach of inlining functions and smaller classes. Larger classes and functions can still live on dedicated pages.
>
> This change is subjective. Feel free to push back if you think any particular interface should be a dedicated page!

> e.g. setting :show-inheritance: etc because each class gets its own subpage.

That's a great point about `:show-inheritance:`. I will open a PR to fix this in every addon. Otherwise, the template and `.. autoclass::` should be very similar - there won't be the `methods` and `attributes` headers, but it will otherwise be the same.